src/core/QuickActionCard/__tests__/QuickActionCard.test.tsx

```diff
@@ -0,0 +1,28 @@
+import { screen } from "@testing-library/dom";
+import { cleanup, render } from "@testing-library/react";
+import React from "react";
+import { LitmusThemeProvider } from "../../..";
+import { QuickActionCard } from "../QuickActionCard";
+import { quickActionTestData } from "../testData";
+
+afterEach(cleanup);
+jest.useFakeTimers();
+
+describe("EditablText component", () => {
+  it("Renders", () => {
+    render(
+      <LitmusThemeProvider platform="litmus-portal">
+        <QuickActionCard
+          quickActions={quickActionTestData}
+          title={"Quick Actions"}
+        />
+      </LitmusThemeProvider>
+    );
+
+    // Get QuickActionCard component
+    const quickActionCardComponent = screen.getByTestId(
+      "quickActionCardComponent"
+    );
+    expect(quickActionCardComponent).toBeTruthy();
```

Comment:
in the test case check if the image is showing up or not and text is there or not.
```
const image = getByAltText('the_alt_text');

    expect(image.src).toContain('the_url');
```
the code will be something like this for the image...(approx)